cmd/benchmark.go

```diff
@@ -186,12 +186,15 @@ func (raw rawBenchmarkCmdArgs) appendVirtualDir(target, virtualDir string) (stri
 
 	switch InferArgumentLocation(target) {
 	case common.ELocation.Blob():
-		p := azblob.NewBlobURLParts(*u)
+		p, err := blob.ParseURL(target)
+		if err != nil {
+			return "", fmt.Errorf("error parsing the url %s. Failed with error %s", target, err.Error())
+		}
 		if p.ContainerName == "" || p.BlobName != "" {
 			return "", errors.New("the blob target must be a container")
 		}
 		p.BlobName = virtualDir
-		result = p.URL()
+		return result.String(), err
```

Comment:
'result' is not used in blob.ParseURL(), is it correct variable to return?

Reply:
fixed